qa/tasks/ceph_manager.py

```diff
@@ -1955,6 +1955,21 @@ def get_osd_dump(self):
         """
         return self.get_osd_dump_json()['osds']
 
+    def get_osd_metadata_json(self):
+        """
+        osd metadata --format=json converted to a python object
+        :returns: the python object containing osd metadata information
+        """
+        out = self.raw_cluster_cmd('osd', 'metadata', '--format=json')
+        return json.loads('\n'.join(out.split('\n')[1:]))
+
+    def get_osd_metadata(self):
```

Comment:
Can we delete this method and rename `get_osd_metadata_json` to `get_osd_metadata`?